### ISSUE TITLE
Attempting to add youtube_username but it isn't working

### ISSUE THREAD

**Issue by p2635** [jekyll/minima#298]
Hi all, this is more a support request but can anyone help me? I'm trying to add the youtube_username to my config but it's not working: https://github.com/p2635/p2635.github.io.

**Comment by p2635** [comment]
After close inspection, I think I'm missing the assets so I tried to simply copy and paste that into my local dir and then tried to do jekyll serve but got this:

`  Conversion error: Jekyll::Converters::Scss encountered an error while converting '/assets/main.scss':
                    File to import not found or unreadable: minima. Load path: /home/user/dir/p2635.github.io/_sass on line 1
`

**Comment by p2635** [comment]
> If you just want to use the theme, I suggest you use the gem-based version. I opened a PR on your repo.

That's amazing, thank you so much for your help. I did try to do a gem installation actually but that didn't work either so I don't know what I did wrong. Somehow you fixed it! I am a newbie at this still but maybe I will check out in detail what you did later. I actually wanted to reject one of the changes (the favicon one) but I didn't know how, I was going to leave it that way **so I know I need to add a favicon **later**. Merci!
